src/DocSearch.tsx

```diff
@@ -39,6 +39,7 @@ export interface DocSearchProps {
    * In the previous example, that would be `hello`.
    */
   debounceDuration?: number | false;
+  checkCompositionEvent?: boolean;
```

Comment:
could we add some docs here to explain what it does? and default value

Reply:
OK. Just added the comment.